lib/calendar_interval.ex

```diff
@@ -204,33 +228,89 @@ defmodule CalendarInterval do
   """
   @spec parse!(String.t()) :: t()
   def parse!(string) do
+    [string, calendar] = parse_including_calendar(string)
+
     case String.split(string, "/", trim: true) do
       [string] ->
-        {ndt, precision} = do_parse!(string)
+        {ndt, precision} = do_parse!(string, calendar)
         new(ndt, precision)
 
       [left, right] ->
         right = String.slice(left, 0, byte_size(left) - byte_size(right)) <> right
-        right = parse!(right)
-        left = parse!(left)
+        right = parse!(right <> " " <> inspect(calendar))
+        left = parse!(left <> " " <> inspect(calendar))
         new(left.first, right.last, left.precision)
     end
   end
 
+  defp parse_including_calendar(string) do
```

Comment:
Yes, it is so a calendar module can be specified.   The `inspect` part of of the proposal was accepted and is in the master branch for Elixir 1.10. I'll check if the sigils were accepted - I can't remember for the moment.

Reply:
yeah, the inspect part was accepted but looks like the sigil not yet.

Regarding inspect, I like that it works by being able to implement `inspect_date` etc callbacks on the calendar implementation.

Should we follow that approach here as well - there would be a similar  `inspect_interval` callback? This might be useful for iso week calendar I guess?